decorator.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'nameable'
+class Decorator < Nameable
+  attr_reader :nameable
+
```

Comment:
- [ ] The `namable` should have both `getter` and `setter`, since we're creating a unique instance whenever the `Decorator` class is initialized